templates/taxi/car_list.html

```diff
@@ -1,7 +1,10 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>
+    Car list
+    <a style="float: right" href="{% url 'taxi:car-create' %}" class="btn btn-primary">CREATE</a>
```

Comment:
Thanks, changed. As for other styling - of course it would be better to apply different styles in css, however styling is out of scope of this task, so I didn't pay much attention to it)